themes/default/content/blog/announcing-community-slack-archive/index.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Announcing Community Slack Archive"
+date: 2022-05-01T15:00:49-07:00
+meta_desc: Pulumi's community slack is now publicly archived at archive.pulumi.com
+meta_image: meta.png
+authors:
+    - lee-briggs
+tags:
+    - community
+---
+
+Creating a place for the Pulumi community to gather, ask questions, get help in real time and share successes has been an important part of the explosive growth we've seen in both users and customers. The Pulumi community slack has grown to over 7000 members and well over 200,000 messages.
+
+Within those 200,000 messages is years of information is kept behind a "walled garden" that is undiscoverable outside Slack's search capabilities.
+
+Today we're announcing that our community slack is now pubicly available via [https://archive.pulumi.com](https://archive.pulumi.com) thanks to [linen](https://linen.dev)
+
+<!--more-->
+
+For many, Slack has become to de-facto collaboration tool. Adding Slack workspaces is relatively straightforward and means you can communicate with various different communities and your colleagues in real time.
+
+This ability to use the tools you're using in your every day work to also quickly get help with Pulumi has proven to be extremely successful. New users tell us they join the Pulumi community Slack because they don't need to add new applications (for example, an IRC client) and because its easy to switch workspaces to engage with our active community.
```

Comment:
```suggestion
This ability to use the tools you're already using and to quickly get help with Pulumi has proven to be extremely successful. New users tell us they join the Pulumi community Slack because they don't need to add new applications (for example, an IRC client) and because it's easy to switch workspaces to engage with our active community.
```

i would suggest dropping the last sentence